test/spec/java_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "Java" do
 
-  ["1.7", "1.8", "8", "11", "13", "15", "16", "17", "18", "11.0.15", "openjdk-11.0.15", "zulu-11.0.15"].each do |jdk_version|
+  ["1.7", "1.8", "8", "11", "13", "15", "16", "17", "18", "11.0.15", "openjdk-11.0.15", "zulu-11.0.15", "heroku-17", "zulu-17"].each do |jdk_version|
```

Comment:
My brain had a confusing moment when first reading "heroku-17" -> "...but that's not a valid Heroku stack?!" 😆